### PR TITLE
Final contribution review in fuzzer section

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Checklist
- [x] All tests are passing
- [x] README / documentation was extended, if necessary
- [x] Used meaningful commit messages

### Description

This is part three of an extensive review of the contribution docs in an attempt to make it easier for new developers to get started with contributing to Solidity, 

In an attempt to make smaller PRS:

- part 1 https://github.com/ethereum/solidity/pull/5120
- part 2 https://github.com/ethereum/solidity/pull/5121

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -222,7 +222,7 @@ If the instrumentation messages did not appear, try switching the cmake flags po
     cmake .. -DCMAKE_C_COMPILER=path/to/afl-clang -DCMAKE_CXX_COMPILER=path/to/afl-clang++
     make solfuzzer
 
-Othwerise, upon execution the fuzzer will halt with an error saying binary is not instrumented:
+Othwerise, upon execution the fuzzer halts with an error saying binary is not instrumented:
```
Comment (axic) [inline docs/contributing.rst]: Othwerise -> Otherwise

File: docs/contributing.rst [hunk 2]
```diff
@@ -259,7 +259,7 @@ from the documentation or the other tests:
 
 The AFL documentation states that the corpus (the initial input files) should not be
 too large. The files themselves should not be larger than 1 kB and there should be
-at most one input file per functionality, so better start with a small number of
+at most one input file per functionality. There is also a tool called ``afl-cmin`` that can trim input files
```
Comment (axic) [inline docs/contributing.rst]: I think these are two different comments.

File: docs/contributing.rst [hunk 3]
```diff
@@ -184,14 +184,14 @@ Running the Fuzzer via AFL
 ==========================
 
 Fuzzing is a technique that runs programs on more or less random inputs to find exceptional execution
-states (segmentation faults, exceptions, etc). Modern fuzzers are clever and do a directed search
+states (segmentation faults, exceptions, etc). Modern fuzzers are clever and run a directed search
 inside the input. We have a specialized binary called ``solfuzzer`` which takes source code as input
 and fails whenever it encounters an internal compiler error, segmentation fault or similar, but
-does not fail if e.g. the code contains an error. This way, internal problems in the compiler
+does not fail if e.g., the code contains an error. This way, fuzzing tools can find internal problems in the compiler.
```
Comment (chriseth) [inline docs/contributing.rst:205]: syntax error.

File: docs/contributing.rst (hunk 2 above)
Comment (chriseth) [reply docs/contributing.rst]: I would not recommend it, we do not know if it works for our case.

**Review by chriseth — APPROVED** [review]

**Pull request merged**